assets/apps/customizer-controls/src/typeface/Typeface.js

```diff
@@ -79,7 +79,7 @@ const Typeface = (props) => {
 
 	useEffect(() => {
 		window.wp.customize.bind('change', (setting) => {
-			if (setting.id === fontFamilyControl) {
+			if (setting && setting.id === fontFamilyControl) {
```

Comment:
@cristian-ungureanu At a first glance, I think this callback would not get executed if the setting wouldn't exist as it's bound to its change.

Did it throw an error before?

Reply:
@cristian-ungureanu this was a proper implementation in the customizer context, but it fails in the Storybook context because of the mock object. 

I think we should rather make sure that the mock of this method doesn't execute the callback inside Storybook (as it shouldn't). We can simply remove the callback execution here: 

https://github.com/codeinwp/neve/blob/439f21dabe1b9e179ddc99d86eb3279f3c027269/.storybook/preview.js#L20